Gems/Atom/RHI/Metal/Code/Source/RHI/ImageView.cpp

```diff
@@ -55,8 +55,8 @@ namespace AZ
             
             //Create a new view if the format, mip range or slice range has changed
             if( isViewFormatDifferent ||
-                levelRange.length != mtlTexture.mipmapLevelCount ||
-                sliceRange.length != textureLength)
+                levelRange.location != 0 || levelRange.length != mtlTexture.mipmapLevelCount ||
```

Comment:
Wouldn't the newly added checks (levelRange.location != 0 and sliceRange.location != 0) mean that we will always create a new view if m_mipLevels or m_arraySize are not 0?